.flox/env/manifest.lock

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
I like the idea but I'm a bit concerned about the maintenance because I'm not familiar with Flox. For example, how was this file generated and how is it kept up-to-date? A little more information in the commit message would be nice.